Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -353,9 +353,10 @@ void CVolumeDirectory::BuildFST()
   m_fst_data.clear();
 
   File::FSTEntry rootEntry = File::ScanDirectoryTree(m_root_directory, true);
-  u32 name_table_size = ComputeNameSize(rootEntry);
+  u32 name_table_size = Common::AlignUp(ComputeNameSize(rootEntry), 1ull << m_address_shift);
```

Comment:
I think this has a 3/4 chance (for Wii games) of triggering the `_dbg_assert_(DVDINTERFACE, name_offset == name_table_size)` further down. Would it be fine if we only did this `AlignUp` when calling `Write32`, so it won't affect the `_dbg_assert_` and the size of `m_fst_data`?